mvc/Tests/Security/SSOListenerTest.php

```diff
@@ -83,7 +83,7 @@ public function testGetPreAuthenticatedData()
 
         $userId = 123;
         $passwordHash = md5('password');
-        $legacyUser = new eZUser(array('contentobject_id' => $userId));
+        $legacyUser = @new eZUser(array('contentobject_id' => $userId));
```

Comment:
Same comment as above. Better manipulate php.ini to silent `E_DEPRECATED` before start.